lib/cfxxl.ex

```diff
@@ -36,6 +36,7 @@ defmodule CFXXL do
   @scan_opts [:ip, :timeout, :family, :scanner]
   @sign_opts [:hosts, :subject, :serial_sequence, :label, :profile, :bundle]
 
+  @spec authsign(CFXXL.Client.t(), String.t(), String.t(), keyword) :: {:ok, any} | {:error, any}
```

Comment:
General style guideline: we always use parentheses after types in typespecs, so: `keyword()`, `any()` and so on, please fix it everywhere